Makefile

```diff
@@ -62,8 +68,8 @@ format:
 	@$(REPO_ROOT)/hack/format.sh $(EFFECTIVE_DIRECTORIES)
 
 .PHONY: check
-check:
-	@$(REPO_ROOT)/hack/check.sh --golangci-lint-config=./.golangci.yaml $(EFFECTIVE_DIRECTORIES)
```

Comment:
where/which is now the configuration been used for golangci-lint?

Reply:
It's the configuration at the root by default called `.golangci.yaml`.

Reply:
hmm... ok... @frewilhelm what do you think, shall we move that config also under: `.github/config`? Since it's used by `make check` which runs `golangci-lint` I'm not sure about it.

Reply:
Hm, I would move `.golangci.yaml` to `.github/config` because (1) the root-repository should look less cluttered, (2) we moved all other config-files there, (3) and `golangci-lint` is primarily used in our GitHub-Actions. One can use it locally, but that is not the main purpose.

Reply:
Sure, I can move the config there. No problem.

Reply:
Done. Moved the config.